### PR TITLE
React props - Cody Epstein

### PR DESCRIPTION
**Pull request by kepsteen**
image

### CODE REVIEW

File: react-props/react-props-and-expressions-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What are props in React?
+
+- How do you declare and access props in a component?
+
+- How do you pass props to a component?
+
+- How do you write JavaScript expressions in JSX?
+
```
Comment (shawnkost) [inline react-props/react-props-and-expressions-quiz-notes.md]: Missing the quiz questions

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just need to answer the quiz questions

File: react-props/react-props-and-expressions-quiz-notes.md (hunk 1 above)
Comment (kepsteen) [reply react-props/react-props-and-expressions-quiz-notes.md]: oops sorry 😅

File: react-props/react-props-and-expressions-quiz-notes.md [hunk 2]
```diff
@@ -6,12 +6,20 @@ Answer the following questions in the provided markdown file before turning in t
 
 - What are props in React?
 
+It is an object which allows for the passing of arguments to react components
+
 - How do you declare and access props in a component?
 
+You declare the props using the type keyword and access them using object destructuring in the parameter of the component definition
```
Comment (shawnkost) [inline react-props/react-props-and-expressions-quiz-notes.md:13]: The type keyword is how we annotate or type our props, but how do we actually declare them?

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**